### PR TITLE
add openexr pinning

### PR DESCRIPTION
**Pull request by hmaarrfk**
Since this has a minor version, it is causing me headaches in builds that built with different pinnings.

Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jakirkham** [comment]
How did we come to this pinning? Also what does the API/ABI compatibility look like for `openexr`?

**Comment by hmaarrfk** [comment]
Openexr has a run exports that pins the minor version.

2.5 is the latest and hottest version that has been included in some builds
2.4 is the previous version

**Comment by mariusvniekerk** [comment]
according to https://abi-laboratory.pro/?view=timeline&l=openexr it broke frequently in the past

**Comment by hmaarrfk** [comment]
Thanks!

**Comment by wolfv** [comment]
looks good to me!

**Comment by wolfv** [comment]
what do we have to wait for to get this merged?

**Comment by mariusvniekerk** [comment]
@hmaarrfk @wolfv @jakirkham Is this safe to merge now?

**Comment by jakirkham** [comment]
Are we happy with the [`run_exports` in the feedstock]( https://github.com/conda-forge/openexr-feedstock/blob/2f973a6babb5612ec8ebdf08250b78a717ae6bb5/recipe/meta.yaml#L16-L18 )?

**Comment by jakirkham** [comment]
> according to https://abi-laboratory.pro/?view=timeline&l=openexr it broke frequently in the past

Asked about getting some more recent versions in the tracker in issue ( https://github.com/lvc/abi-tracker/issues/13 ).

**Comment by hmaarrfk** [comment]
Thanks for asking upstream.

New data shows good consistency among `2.5.X` releases.

**Comment by jakirkham** [comment]
Ofc!

Assuming we start with 2.5.1, yes.

**Comment by wolfv** [comment]
should we press the merge button?

File: recipe/conda_build_config.yaml
Comment (jakirkham) [inline recipe/conda_build_config.yaml:545]: Would suggest add `.1` due to the SONAME change issue between 2.5.0 and 2.5.1

```suggestion
openexr:
  - 2.5.1
```

**Review by jakirkham — COMMENTED** [review]
One suggestion below. Otherwise LGTM.

**Comment by hmaarrfk** [comment]
Thansk @jakirkham

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Mark! 😄

**Comment by hmaarrfk** [comment]
hmm this really should have used a migrator.

Because some packages were built with openexr 2.5.2, this pinning is breaking certain packages. this should have really used a migrator.

**Comment by CJ-Wright** [comment]
you are more than welcome to create a migrator PR if you'd like

**Pull request merged**
